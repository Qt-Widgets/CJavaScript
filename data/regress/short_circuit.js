function True () { console.log("true" ); return true ; }
function False() { console.log("false"); return false; }
function Run  () { console.log("run"  ); return true ; }

True();
False();

console.log("True () || Run()", True () || Run());
console.log("False() || Run()", False() || Run());

console.log("True () && Run()", True () && Run());
console.log("False() && Run()", False() && Run());
