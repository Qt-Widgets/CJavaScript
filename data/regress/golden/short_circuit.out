true
false
true
True () || Run() true
false
run
False() || Run() true
true
run
True () && Run() true
false
False() && Run() false
